apps/backend/src/modules/surveys/survey.repository.ts

```diff
@@ -0,0 +1,66 @@
+import { RelationName } from "~/libs/enums/enums.js";
+import { type Repository } from "~/libs/types/types.js";
+
+import { OnboardingAnswerEntity } from "./onboarding/onboarding-answer.entuty.js";
+import { type OnboardingAnswerModel } from "./onboarding/onboarding-answer.model.js";
+import { OnboardingQuestionEntity } from "./onboarding/onboarding-question.entity.js";
+import { type OnboardingQuestionModel } from "./onboarding/onboarding-question.model.js";
+
+class SurveyRepository implements Repository {
+	private onboardingAnswerModel: typeof OnboardingAnswerModel;
+	private onboardingQuestionModel: typeof OnboardingQuestionModel;
+
+	public constructor(
+		onboardingAnswerModel: typeof OnboardingAnswerModel,
+		onboardingQuestionModel: typeof OnboardingQuestionModel,
+	) {
+		this.onboardingAnswerModel = onboardingAnswerModel;
+		this.onboardingQuestionModel = onboardingQuestionModel;
+	}
+
+	public create(): ReturnType<Repository["create"]> {
+		return Promise.resolve(null);
+	}
+
+	public delete(): ReturnType<Repository["delete"]> {
+		return Promise.resolve(true);
+	}
+
+	public find(): ReturnType<Repository["find"]> {
+		return Promise.resolve(null);
+	}
+
+	public findAll(): ReturnType<Repository["findAll"]> {
+		return Promise.resolve([]);
+	}
```

Comment:
Implement those as well, we shouldn't have any placeholders

Reply:
Maybe I needed to change placeholders to something like:

 `public delete(): Promise<boolean> {
        throw new Error(ErrorMessage.NOT_IMPLEMENTED);
    }`
    
 But I've already done it